lib/tasks/claims.rake

```diff
@@ -143,9 +143,7 @@ namespace :claims do
   end
 
   desc 'Create a report of submitted claims'
-  task :submitted_claims_report, [:filename] => :environment do |_task, args|
-    require_relative 'rake_helpers/submitted_claims'
-
-    RakeHelpers::SubmittedClaims.write args[:filename]
+  task submitted_claims_report: :environment do |_task, args|
+    StatsReportGenerationJob.perform_later('submitted_claims')
   end
```

Comment:
I don't think we need this anymore, if we are not using the cronjob?!

Reply:
Probably not.